dialogs.hpp

```diff
@@ -2115,14 +2115,24 @@ class HQ_button_Gstatic: RscButton
 };
 class HQ_button_Gremove: RscButton
 {
-	idc = 106;
+	idc = 109;
 	text = "Helicopter"; //--- ToDo: Localize;
-	x = 0.37749 * safezoneW + safezoneX;
+	x = 0.482498 * safezoneW + safezoneX;
 	y = 0.415981 * safezoneH + safezoneY;
 	w = 0.175015 * safezoneW;
 	h = 0.0560125 * safezoneH;
 	action = "closeDialog 0;nul = [civHeli] call addFIAveh;";
 };
+class HQ_button_offroad: RscButton
+{
+	idc = 105;
+	text = "Boat"; //--- ToDo: Localize;
+	x = 0.272481 * safezoneW + safezoneX;
+	y = 0.415981 * safezoneH + safezoneY;
+	w = 0.175015 * safezoneW;
+	h = 0.0560125 * safezoneH;
+	action = "closeDialog 0;nul = execVM ""\REINF\buyBoat.sqf"";";
```

Comment:
It says \reinf\buyboat doesn't exist, i think there is a typo, have to check again

Reply:
found the typo... the first \ shouldn't be there